cmake/ConfigureCompiler.cmake

```diff
@@ -217,6 +217,12 @@ else()
       -Wno-undefined-var-template
       -Wno-tautological-pointer-compare
       -Wno-format)
+    set(USE_AVX ON CACHE BOOL "Enable AVX instructions")
+    if (USE_AVX)
+      add_compile_options(-mavx)
+    else()
+      add_compile_options(-msse4)
```

Comment:
I am pretty sure `-msse4` will also break correctness.

Reply:
I removed the -msse4 option so that there's an easy way to disable all the vector instructions (just in case).